Microsoft.Research/RegressionTest/ClousotTests/v.cs

```diff
@@ -48,6 +50,1516 @@ public TestContext TestContext
+            contractReferenceAssemblies: true,
+            exe: false,
+            compiler: "CS"),
+        new Options(
+            sourceFile: @"Microsoft.Research\RegressionTest\ClousotTests\Sources\ArrayForAll.cs",
+            compilerOptions: "/optimize",
+            libPaths: new string[0],
+            references: new[] { "System.Core.dll" },
+            options: "-infer autopropertiesensures -suggest requires -nonnull -bounds:noobl -arrays -show progress",
+            contractReferenceAssemblies: true,
+            exe: false,
+            compiler: "CS"),
+        new Options(
+            sourceFile: @"Microsoft.Research\RegressionTest\ClousotTests\Sources\TypeSpecializations.cs",
+            compilerOptions: "/optimize",
+            libPaths: new string[0],
+            references: new[] { "" },
+            options: "-infer autopropertiesensures -suggest requires -nonnull:noobl -bounds:noobl -show progress -show validations",
+            contractReferenceAssemblies: true,
+            exe: false,
+            compiler: "CS"),
+        new Options(
+            sourceFile: @"Microsoft.Research\RegressionTest\ClousotTests\Sources\HeapCrash.cs",
+            compilerOptions: "/optimize /r:System.Configuration.dll",
+            libPaths: new string[0],
+            references: new[] { "" },
+            options: "-infer autopropertiesensures -suggest requires -nonnull -bounds:noobl -arrays -show progress",
+            contractReferenceAssemblies: true,
+            exe: false,
+            compiler: "CS"),
+        new Options(
+            sourceFile: @"Microsoft.Research\RegressionTest\ClousotTests\Sources\ForAll.cs",
+            compilerOptions: "/optimize",
+            libPaths: new string[0],
+            references: new[] { "System.Core.dll" },
+            options: "-infer autopropertiesensures -suggest requires -nonnull -bounds:noobl -arrays -show progress",
+            contractReferenceAssemblies: true,
+            exe: false,
+            compiler: "CS"),
+        new Options(
+            sourceFile: @"Microsoft.Research\RegressionTest\ClousotTests\Sources\Mulder.cs",
+            compilerOptions: "/optimize",
+            libPaths: new string[0],
+            references: new[] { "" },
+            options: "-infer autopropertiesensures -suggest requires -nonnull -bounds:noobl -show progress",
+            contractReferenceAssemblies: true,
+            exe: false,
+            compiler: "CS"),
+        new Options(
+            sourceFile: @"Microsoft.Research\RegressionTest\ClousotTests\Sources\IOperations.cs",
+            compilerOptions: "",
+            libPaths: new string[0],
+            references: new[] { "" },
+            options: "-infer autopropertiesensures -suggest requires -bounds -show:validations;unreached",
+            contractReferenceAssemblies: false,
+            exe: false,
+            compiler: "CS"),
+#if false
+  new Options(
```

Comment:
:memo: I did some simple find/replace below this marker, but they haven't been completely converted yet. Now that I have a better idea what the output should look like, I'll do the final conversion using an XSLT.